doc/sphinxext/gallery_generator.py

```diff
@@ -240,9 +240,13 @@ def thumbfilename(self):
 
     @property
     def apiname(self):
-        name = self.modulename.split("_")
-        name = name[1::]
-        return "_".join(name)
+        name=""
```

Comment:
Maybe just return None if nothing is found?

Reply:
Okay, makes sense.